src/libs/actions/IOU.ts

```diff
@@ -3087,12 +3087,10 @@ function deleteMoneyRequest(transactionID: string, reportAction: OnyxTypes.Repor
             onyxMethod: Onyx.METHOD.MERGE,
             key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${chatReport?.reportID}`,
             value: {
-                [reportPreviewAction?.reportActionID ?? '']: shouldDeleteIOUReport
```

Comment:
why are we removing this `shouldDeleteIOUReport` check here?

Reply:
@grgia Currently, if `shouldDeleteIOUReport` we will delete PREVIEW report action in the main chat (1:1 chat). With this update, we will not remove PREVIEW report action from the report action list, we only add `deleted` field as a flag to mark this request is deleted

Reply:
Yeah this is just a change to stop deleting the report preview action in the `successData` for `DeleteMoneyRequest`